remo/base/views.py

```diff
@@ -41,7 +42,10 @@ def login_failure(self, msg=''):
 def main(request):
     """Main page of the website."""
     featured_rep = utils.latest_object_or_none(FeaturedRep)
-    return render(request, 'main.jinja', {'featuredrep': featured_rep})
+    feed = feedparser.parse('http://planet.mozillareps.org/rss20.xml')
```

Comment:
Please use something like `settings.PLANET_URL` and add the actual url in the project's base settings.